examples/ex1_plot_map.py

```diff
@@ -0,0 +1,45 @@
+"""
+=======================
+Three dimensional plots
+=======================
+sunpy can interface with the `pyvista` package to produce interactive 3D plots.
```

Comment:
```suggestion

Using sunkit-pyvista, one can interface with the `pyvista` package to produce interactive 3D plots for sunpy Maps.
```
Probably want to link the sunpy map docs, tho I think we will need to make sure its setup in the docs/conf.py file.

Reply:
That'll require `"sunpy": ("https://docs.sunpy.org/en/stable/", None)` to be added to as inter sphinx references right?

Reply:
Yeah.